deepchem/molnet/load_function/kt_dataset.py

```diff
@@ -0,0 +1,210 @@
+  The raw data csv file contains columns below:
+  Please refer to https://arxiv.org/pdf/2008.08078 (2020) for which the dataset was hand curated. Benchmark training is also present in this work. Accuracy levels (rmse) reported here are of the order 0.01  for alpha and beta on unseen data. 
+   References
+  ----------
+  [1] Marcus, Yizhak. "The properties of organic liquids that are relevant to their use as solvating solvents." Chemical Society Reviews 22, no. 6 (1993): 409-416.
+
+
+  Loading ktdataset.
+
+  - "csmiles" - canonical SMILES representation of the molecular structure
+  - "cid" - PubChem CID of molecules
+  - "alpha" - KT paramter that quantifies the acidity of a molecule
+  - "beta" - KT paramter that quantifies the basicity of a molecule
+  
+  Parameters
+  ----------
+  featurizer : {List of allowed featurizers for this dataset}
+    A featurizer that inherits from deepchem.feat.Featurizer.
+  transformers : List{List of allowed transformers for this dataset}
+    A transformer that inherits from deepchem.trans.Transformer.
+  splitter : {List of allowed splitters for this dataset}
+    A splitter that inherits from deepchem.splits.splitters.Splitter.
+  reload : bool (default True)
+    Try to reload dataset from disk if already downloaded. Save to disk
+    after featurizing.
+  data_dir : str, optional
+    Path to datasets.
+  save_dir : str, optional
+    Path to featurized datasets.
+  featurizer_kwargs : dict
+    Specify parameters to featurizer, e.g. {"size": 1024}
+  splitter_kwargs : dict
+    Specify parameters to splitter, e.g. {"seed": 42}
+  transformer_kwargs : dict
+    Maps transformer names to constructor arguments, e.g.
+    {"BalancingTransformer": {"transform_x":True, "transform_y":False}}
+  **kwargs : additional optional arguments.
+
+  Returns
+  -------
+  tasks, datasets, transformers : tuple
+    tasks : list
+      Column names corresponding to machine learning target variables.
+    datasets : tuple
+      train, validation, test splits of data as
+      ``deepchem.data.datasets.Dataset`` instances.
+    transformers : list
+      ``deepchem.trans.transformers.Transformer`` instances applied
+      to dataset.
+
+  References
+  ----------
+  MLA style references for the KT parameter dataset. 
+    [1] Marcus, Yizhak. "The properties of organic liquids that are relevant to their use as solvating solvents." Chemical Society Reviews 22.6 (1993): 409-416.
+  Examples
+  --------
+  >> import deepchem as dc
+  >> tasks, datasets, transformers = dc.molnet.load_kt_dataset(reload=False)
+  >> train_dataset, val_dataset, test_dataset = datasets
+  >> n_tasks = len(tasks)
+  >> n_features = train_dataset.get_data_shape()[0]
+  >> model = dc.models.MultitaskClassifier(n_tasks, n_features)
+"""
+
```

Comment:
Can you remove this whitespace?

Reply:
Thanks. I have also run yapf to check if formatting is fine. It loads as expected without errors on testing with my local copy.